.travis.yml

```diff
@@ -17,7 +18,24 @@ before_script:
   - composer selfupdate --quiet
   - composer install --prefer-dist --no-interaction --no-progress
   - ./vendor/bin/phpunit
-  - composer update --prefer-dist --no-interaction --no-progress
 
 script:
-  - ./vendor/bin/phpunit
+  - make test
+
+after_script:
```

Comment:
yeah, you may be right. Also we may have to add the `skip_cleanup: true` option so the artefact is not stashed, right?